setup.py

```diff
@@ -23,7 +23,8 @@
           'inflection==0.5.1',
           'joblib==1.0.1',
           'numpy<1.21.0',
-          'python-dateutil==2.8.2'
+          'python-dateutil==2.8.2',
```

Comment:
Can we remove this dependency here, it's redundant since it comes with ppw-singer-python https://github.com/transferwise/pipelinewise-singer-python/blob/master/setup.py

Reply:
thanks, removed by 7c42a7d424dd9cf5a44f98c785578cf698ec2da8